apps/server/src/shared/common/utils/transaction.util.ts

```diff
@@ -0,0 +1,15 @@
+import { EntityManager } from '@mikro-orm/core';
+import { Injectable } from '@nestjs/common';
+import { Logger } from '@src/core/logger';
+
+@Injectable()
+export class TransactionUtil {
+	constructor(private readonly _em: EntityManager, private readonly logger: Logger) {}
+
+	async doTransaction(fn: () => Promise<void>): Promise<void> {
```

Comment:
we spoke about transactions in person, please remove for now.

Reply:
done